tidb-lightning.toml

```diff
@@ -153,6 +163,21 @@ backslash-escape = true
 # if a line ends with a separator, remove it.
 trim-last-separator = false
 
+# file level routing rule that map file path to schema,table,type,sort-key
+# The schema, table , type and key can be either a constant string or template strings
+# supported by go regexp. 
+#[[mydumper.files]]
+# pattern is  a regexp in Go syntax for matching the file path.
+# pattern = "(?i)^(?:[^/]*/)(?P<schema>[^/.]+)\.([^/.]+)(?:\.([0-9]+))?\.(sql|csv)$"
```

Comment:
```suggestion
# pattern = '(?i)^(?:[^/]*/)(?P<schema>[^/.]+)\.([^/.]+)(?:\.([0-9]+))?\.(sql|csv)$'
```

use single quotes to avoid `\` escaping stuff